### PR TITLE
Use cml to create download link PR

### PR DESCRIPTION
**Pull request by dacbd**
As part of an effort to use our own tools more/where appropriate cml can create the PR for the download links.

### CODE REVIEW

**Review by DavidGOrtega — APPROVED** [review]
do it 😎

**Review by yathomasi — COMMENTED** [review]
I also had a thought. Thanks for implementing this. 

I just want to confirm that, Will cml catch the token by passing it as an argument as I don't see it https://cml.dev/doc/ref/pr as an option and we previously pass it as an env secret?

**Comment by rogermparent** [comment]
Great stuff! The implementation is surprisingly elegant looking as well. Reminds me that we still need to search and destroy all `create-pull-request` action usages (and update our current CML usages IIRC unless it's been handled already).

> I just want to confirm that, Will cml catch the token by passing it as an argument as I don't see it https://cml.dev/doc/ref/pr as an option and we previously pass it as an env secret?

The docs are in https://cml.dev/doc/ref#generic-options, on its own page since all commands accept `--token`. Hopefully the Action-supplied `GITHUB_TOKEN` works without a hitch but it's been doing well recently.

File: content/docs/user-guide/data-management/cloud-versioning.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ When the `version_aware` option is enabled on a `dvc remote`:
 
 Note that when `version_aware` is in use, DVC does not delete current versions
 or restore noncurrent versions of objects in cloud storage. So the current
-version of an object in cloud storage may not match the version of a file in your DVC repository.
+version of an object in cloud storage may not match the version of a file in
+your DVC repository.
```
Comment (rogermparent) [inline content/docs/user-guide/data-management/cloud-versioning.md:79]: huh, what happened here?

**Review by rogermparent — APPROVED** [review]

**Comment by dacbd** [comment]
> I also had a thought. Thanks for implementing this.
> 
> I just want to confirm that, Will cml catch the token by passing it as an argument as I don't see it https://cml.dev/doc/ref/pr as an option and we previously pass it as an env secret?

@yathomasi that is correct, this part is documented here (which might be a bit out of the way) https://cml.dev/doc/ref

**Comment by dacbd** [comment]
> Great stuff! The implementation is surprisingly elegant looking as well. Reminds me that we still need to search and destroy all `create-pull-request` action usages (and update our current CML usages IIRC unless it's been handled already).
> 
> > I just want to confirm that, Will cml catch the token by passing it as an argument as I don't see it https://cml.dev/doc/ref/pr as an option and we previously pass it as an env secret?
> 
> The docs are in https://cml.dev/doc/ref#generic-options, on its own page since all commands accept `--token`. Hopefully the Action-supplied `GITHUB_TOKEN` works without a hitch but it's been doing well recently.

I am in the progress of hunting down all of these 👍 

The default action supplied token should be sufficient, I'll note that there is a github repo option that could break this but it can simply be resolve by including explicit token permissions like here:
https://github.com/iterative/cml-playground/blob/94430875f7a2b0e8e24537b382cf52d68268a564/.github/workflows/cml-1104.yml#L11-L14

File: content/docs/user-guide/data-management/cloud-versioning.md (hunk 1 above)
Comment (yathomasi) [reply content/docs/user-guide/data-management/cloud-versioning.md:79]: Yes, just a prettier issue. It looks like a PR got merged without a fix and checks were failing here.

**Pull request merged**